examples/configs/staging.bat

```diff
@@ -0,0 +1,19 @@
+@ECHO OFF
+
+:: Neurobooth folders
+setx /m NB_INSTALL C:\neurobooth\
+setx /m NB_CONFIG %USERPROFILE%\.neurobooth_os\
+setx /m NB_ACQ_CONDA C:\Users\ACQ\anaconda3\envs\neurobooth-staging
+setx /m NB_STM_CONDA C:\Users\STM\anaconda3\envs\neurobooth-staging
+setx /m NB_ACQ_DATA D:\neurobooth\neurobooth_data
```

Comment:
There is a data folder for each machine that gets robocopied over after a session. Should consider all of them.

Maybe instead of having a separate variable for each machine, we have a single variable that differs across the three? Which would then mean we have different setup scripts for each machine. Need to think through if this would ultimately reduce or increase complexity.